tests/Finder/Iterator/RealPathFilterIteratorTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Finder\Iterator;
+
+use Infection\Tests\Fixtures\Autoloaded\Finder\MockRelativePathFinder;
+use Infection\Tests\Fixtures\Finder\MockRealPathFinder;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ */
+final class RealPathFilterIteratorTest extends TestCase
+{
+    /**
+     * @dataProvider providesFinders
+     *
+     * @param string $finder
+     * @param int $expectedFilecount
+     */
+    public function test_it_differs_from_relative_path(string $finder, int $expectedFilecount)
```

Comment:
The intention of this test is to prove the difference between our own version and the symfony version.